inst/extdata/fusionreference.txt

```diff
@@ -1989,361 +2258,680 @@ BZW2--GAP43	pancanfus.txt	TCGAFusion
 BZW2--IMMP2L	pancanfus.txt	TCGAFusion
 BZW2--RRM1	pancanfus.txt	TCGAFusion
 C10orf11--ADK	pancanfus.txt	TCGAFusion
+LRMDA--ADK	pancanfus.txt	TCGAFusion
 C10orf11--ADK	pancanfus.txt	TCGAFusion
+LRMDA--ADK	pancanfus.txt	TCGAFusion
 C10orf11--ADK	pancanfus.txt	TCGAFusion
+LRMDA--ADK	pancanfus.txt	TCGAFusion
 C10orf11--ADK	pancanfus.txt	TCGAFusion
+LRMDA--ADK	pancanfus.txt	TCGAFusion
 C10orf11--CYP2C19	pancanfus.txt	TCGAFusion
+LRMDA--CYP2C19	pancanfus.txt	TCGAFusion
 C10orf11--RTKN2	pancanfus.txt	TCGAFusion
+LRMDA--RTKN2	pancanfus.txt	TCGAFusion
 C10orf11--RUFY2	pancanfus.txt	TCGAFusion
+LRMDA--RUFY2	pancanfus.txt	TCGAFusion
 C10orf11--VCL	pancanfus.txt	TCGAFusion
+LRMDA--VCL	pancanfus.txt	TCGAFusion
 C10orf116--SNW1	pancanfus.txt	TCGAFusion
+ADIRF--SNW1	pancanfus.txt	TCGAFusion
 C10orf118--GFRA1	pancanfus.txt	TCGAFusion
+CCDC186--GFRA1	pancanfus.txt	TCGAFusion
 C10orf125--DOCK1	pancanfus.txt	TCGAFusion
+FUOM--DOCK1	pancanfus.txt	TCGAFusion
 C10orf18--ABI1	pancanfus.txt	TCGAFusion
+TASOR2--ABI1	pancanfus.txt	TCGAFusion
 C10orf18--AKR1E2	pancanfus.txt	TCGAFusion
+TASOR2--AKR1E2	pancanfus.txt	TCGAFusion
 C10orf18--NET1	pancanfus.txt	TCGAFusion
+TASOR2--SLC6A2	pancanfus.txt	TCGAFusion
+TASOR2--SLC6A5	pancanfus.txt	TCGAFusion
 C10orf18--NET1	pancanfus.txt	TCGAFusion
+TASOR2--SLC6A2	pancanfus.txt	TCGAFusion
+TASOR2--SLC6A5	pancanfus.txt	TCGAFusion
 C10orf26--ADAMTS2	pancanfus.txt	TCGAFusion
+WBP1L--ADAMTS2	pancanfus.txt	TCGAFusion
 C10orf26--ARL3	pancanfus.txt	TCGAFusion
+WBP1L--ARL3	pancanfus.txt	TCGAFusion
 C10orf26--C6orf105	pancanfus.txt	TCGAFusion
+WBP1L--ADTRP	pancanfus.txt	TCGAFusion
 C10orf26--SFXN2	pancanfus.txt	TCGAFusion
+WBP1L--SFXN2	pancanfus.txt	TCGAFusion
 C10orf26--SUFU	pancanfus.txt	TCGAFusion
+WBP1L--SUFU	pancanfus.txt	TCGAFusion
 C10orf26--SUFU	pancanfus.txt	TCGAFusion
+WBP1L--SUFU	pancanfus.txt	TCGAFusion
 C10orf26--SUFU	pancanfus.txt	TCGAFusion
+WBP1L--SUFU	pancanfus.txt	TCGAFusion
 C10orf26--SUFU	pancanfus.txt	TCGAFusion
+WBP1L--SUFU	pancanfus.txt	TCGAFusion
 C10orf26--SUFU	pancanfus.txt	TCGAFusion
+WBP1L--SUFU	pancanfus.txt	TCGAFusion
 C10orf26--TRIM8	pancanfus.txt	TCGAFusion
+WBP1L--TRIM8	pancanfus.txt	TCGAFusion
 C10orf32--C10orf26	pancanfus.txt	TCGAFusion
+BORCS7--WBP1L	pancanfus.txt	TCGAFusion
 C10orf32--C10orf26	pancanfus.txt	TCGAFusion
+BORCS7--WBP1L	pancanfus.txt	TCGAFusion
 C10orf46--EIF3A	pancanfus.txt	TCGAFusion
+CACUL1--EIF6	pancanfus.txt	TCGAFusion
 C10orf46--IL1RN	pancanfus.txt	TCGAFusion
+CACUL1--IL1RN	pancanfus.txt	TCGAFusion
 C10orf57--CREB3L3	pancanfus.txt	TCGAFusion
+TMEM254--CREB3L3	pancanfus.txt	TCGAFusion
 C10orf68--CCDC7	pancanfus.txt	TCGAFusion
+CCDC7--CCDC7	pancanfus.txt	TCGAFusion
```

Comment:
I noticed these when spot checking the file. A self fusion seems pointless.

Reply:
These are in there because they are sometimes real and indicative of duplication events. If a intragene fusion occurs, it may or may not be an artifact, but as it is being observed in TCGA high-confidence set, then we weight it as being putative oncogenic.